images/bootstrap/Dockerfile

```diff
@@ -99,8 +99,9 @@ RUN  curl -Lo ./bazelisk https://github.com/bazelbuild/bazelisk/releases/downloa
 # Cache the most commonly used bazel versions inside the image
 # and remove resulting cache directories to save a few hundred MB
 RUN USE_BAZEL_VERSION=3.4.1 bazel version && \
-  USE_BAZEL_VERSION=4.0.0 bazel version && \
-  rm -rf /root/.cache/bazel
+    USE_BAZEL_VERSION=4.0.0 bazel version && \
```

Comment:
Should we maybe also prepull 4.1.0 which is used in project-infra IIRC?

Reply:
Good catch, in order to keep the image size low I've changed the PR to only prepull the versions used in kubevirt and project-infra, PTAL